src/components/FlightTicket/FlightTicket.tsx

```diff
@@ -1,24 +1,22 @@
 import styles from "./FlightTicket.module.css";
 import { FlightTicketColorVariant, LegType } from "./FlightTicket.definitions";
+import FlightDetailsModal from "../../modals/FlightDetailsModal/FlightDetailsModal";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import {
   faCircleArrowRight,
   faPlaneArrival,
   faPlaneDeparture,
   faPlane,
 } from "@fortawesome/free-solid-svg-icons";
+import { useState } from "react";
 import type { FlightTicketProps } from "./FlightTicket.definitions";
 
 const FlightTicket = ({
-  date,
-  departingAirport,
-  arrivingAirport,
-  airline,
-  legType,
+  flight,
   colorVariant,
   isLastElement,
 }: FlightTicketProps) => {
-  const renderLegTypeIcon = (legType: LegType) => {
+  const renderLegTypeIcon = (legType: string) => {
```

Comment:
Why is this a string? I agree with the first version where it is a `LegType`

Reply:
This is a string because LegType is a string in the flight-leg interface. We also get a string from the airtable correct?